circle.yml

```diff
@@ -27,6 +27,14 @@ executors:
     environment:
       PLATFORM: linux
 
+  # Docker image with non-root "node" user
+  non-root-docker-user:
+    docker:
+      - image: cypress/base:12.12.0
```

Comment:
Is there a reason not to use the version of Node shipped with Cypress, 12.0.0?

Reply:
I would say use 12.0.0 so it gets bumped whenever we bump the internal Cypress version and nobody has to wonder "is this version 12.12.0 and not 12.0.0 for a specific reason, or is it just like that?" But not that important.

Reply:
done